File: SportsWorldAPI/Interfaces/IAthlete.cs
namespace SportsWorldAPI.Interfaces;

// Definerer egenskaper for Athlete-objekt
interface IAthlete
{
    int Id {get; set; }
    string Name {get; set; }
    string Gender {get; set; }
    double Price {get; set; }
    string Image {get; set;}
    bool PurchaseStatus {get; set;}
}

File: sportsworld/src/interfaces/IAthlete.ts
export interface IAthlete{
    id: number, // satt som obligatorisk fordi TypeScript ikke vil sende "undefined" til en funksjon som krever number(i PurchaseAthlete)
    name: string, 
    gender: string,
    price: number, 
    image: string,
    purchaseStatus: boolean
}